docs/RUNNING_ON_A_BUDGET.md

```
@@ -0,0 +1,98 @@
+
+---
+
+## 5. Token-Saving Best Practices
+
+To prevent unnecessary API costs or hitting rate limits, implement the following practices:
+
+1. **Incremental Batch Processing**:
+   Instead of loading hundreds of job listings into `batch/batch-input.tsv` at once, prepare and evaluate them in smaller chunks of 5-10 jobs. This lets you inspect the output quality before committing to a larger run.
+2. **Use the Dry Run Flag**:
+   Always run a dry run first to verify which offers will be processed:
+   ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)